### DISCUSSION TITLE
멀티플레이 room에 접속한 인원중에 첫번째로 접속했는지 아는 방법이 있을까요??

### DISCUSSION THREAD

**Discussion by dydgnlkim** [naverz/zepeto-studio-kor#1165] [5. Q&A]
멀티플레이시 클라이언트 단에서  room 에 총 플레이어가 몇명있고, 가장 첫번째로 접속한 사람을 서버에서 공유 받고싶은데 어떤 로직으로 찾아야할까요??

**Comment by zlmrzlo** [comment]
서버 코드에서 첫번째로 들어오는 사람을 따로 확인하면 될 것 같아요.

**Reply by dydgnlkim** [reply]
서버 코드에서 첫 번째로 들어오는 사람을 판별 할 수 있는 방법을 코드로 알려주실수있으실까요?

**Reply by zlmrzlo** [reply]
그것과 관련해서 구현된 예제가 있습니다. 제페토에서 제공하고 있는 무빙 블록 예제를 참조하시면 좋을 것 같아요.
https://github.com/naverz/zepeto-studio-kor/discussions/902
